### ISSUE TITLE
it.each node range doesn't include callback body

### ISSUE THREAD

**Issue by tinganho** [jest-community/jest-editor-support#37]
I noticed, that the `it.each` doesn't include the node range for the callback body

See `start` and `end` markers below:
```ts
/*start*/it.each([
            ['some'],
        ])('some', async (some: string) => /*end*/{
// Some code
        });
```

**Comment by seanpoulter** [comment]
Thanks for reporting the issue @tinganho. We've got another issue about `test.each`, #29. That work is currently blocked by PR #30 and we need some help reviewing to get it across the finish line. Join the conversation if you've got the time!
